src/structures/ClientApplication.js

```diff
@@ -23,6 +30,33 @@ class ClientApplication extends Application {
   _patch(data) {
     super._patch(data);
 
+    if ('tags' in data) {
+      /**
+       * The tags this application has (max of 5)
+       * @type {?string[]}
```

Comment:
Since `ClientApplication#tags` is now required:
```suggestion
       * @type {string[]}
```